benchmarks/envoy_filter_fuzz_test/build.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+export CFLAGS="$CFLAGS"
+export CXXFLAGS="$CXXFLAGS"
+
+declare -r FUZZER_TARGETS_CC=$(find . -name *_fuzz_test.cc)
```

Comment:
Envoy is a crazy slow build, so lets optimize here by just specifying the filter_fuzz_test fuzzer file explicitly.

Reply:
+1